File: src/main/groovy/com/dimaslanjaka/gradle/Utils/file/StreamHelper.java
package com.dimaslanjaka.gradle.Utils.file;

import java.io.File;
import java.io.FileInputStream;
import java.io.FileNotFoundException;
import java.io.InputStream;

public class StreamHelper {
	InputStream inputStream;
	String fileName;
	long fileSize;

	public StreamHelper(String fileName) throws FileNotFoundException {
		this.fileName = fileName;
		File file = new File(fileName);
		inputStream = new FileInputStream(file);
		this.fileSize = file.length();
	}
}

File: src/main/kotlin/com/dimaslanjaka/gradle/utils/file/StreamHelper.kt
package com.dimaslanjaka.gradle.utils.file

import java.io.File
import java.io.FileInputStream
import java.io.InputStream

class StreamHelper(var fileName: String?) {
    var inputStream: InputStream?
    var fileSize: Long

    init {
        val file = File(fileName)
        inputStream = FileInputStream(file)
        fileSize = file.length()
    }
}